lib/cinch/plugins/karma.rb

```diff
@@ -29,10 +29,12 @@ def listen(m)
         channel = m.channel.name
 
         # Scan messages for multiple karma items
-        m.message.scan(/(\s|\A)(\w+|\(.+?\))(\+\+|--)(\s|\z)/).each do |k|
+        m.message.scan(/(^|\s|\A|\b)(\w+|\(.+?\))(\+\+|--)(\s|\z|$)/).each do |k|
           process_karma(channel, k[1].gsub(/\(|\)/, '').downcase, k[2])
+	  item = k[1].gsub(/\(|\)/, '')
```

Comment:
Instead of using this regex twice (code duplication is no good), it's probably better to pull `item` out earlier and send the variable to `process_karma`